test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -60,21 +68,54 @@ def test_basic_proxy(self):
             r = http.request("GET", "%s/" % self.https_url)
             assert r.status == 200
 
+    @onlyPy3
     def test_https_proxy(self):
+        with proxy_from_url(self.https_proxy_url, ca_certs=DEFAULT_CA) as https:
+            r = https.request("GET", "%s/" % self.https_url)
+            assert r.status == 200
+
+            r = https.request("GET", "%s/" % self.http_url)
+            assert r.status == 200
+
+    @onlyPy2
+    def test_https_proxy_not_supported(self):
         with proxy_from_url(self.https_proxy_url, ca_certs=DEFAULT_CA) as https:
             r = https.request("GET", "%s/" % self.http_url)
             assert r.status == 200
 
             with pytest.raises(ProxySchemeUnsupported):
```

Comment:
Let's verify the message on the error once, or more if we decide to change the error message mentioned above